### PR TITLE
Modify transaction beat logic to prevent lose queued transactions before timeout

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR is to prevent lose queued transactions before timeout.

Before this PR, queued transactions is dropped in 1 minute, before timeout.
Because, when pending list is empty, beat time also removed.
So when beat time checking logic compares beat time and now time, every 1 minute, the beat time doesn't exist and it calculate with zero time. Therefore the queued transaction will be removed.

In this PR, below logics are modified.

- if beat time is timed out, all queued txs are removed and the beat time is removed.
- if there is no beat time and new transaction is enqueued, the beat time is set.

## Types of changes

Please put an x in the boxes related to your change.

- [x] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: blockchain/tx_pool.go [hunk 1]
```diff
@@ -312,16 +312,21 @@ func (pool *TxPool) loop() {
 			// Handle inactive account transaction eviction
 		case <-evict.C:
 			pool.mu.Lock()
-			for addr := range pool.queue {
+			for addr, beat := range pool.beats {
 				// Skip local transactions from the eviction mechanism
 				if pool.config.KeepLocals && pool.locals.contains(addr) {
+					delete(pool.beats, addr)
```
Comment (aidan-kwon) [inline blockchain/tx_pool.go]: It this line required?

Comment (KimKyungup) [reply blockchain/tx_pool.go]: If these is not this line, the local account's beat can not be removed.

Comment (aidan-kwon) [reply blockchain/tx_pool.go]: I understand. Thank you!

**Comment by KimKyungup** [comment]
@nogang @jeonghonah PTAL. :)

**Pull request merged**